packages/web-components/src/components/content-group-banner/__stories__/content-group-banner.stories.ts

```diff
@@ -0,0 +1,124 @@
+// eslint-disable-next-line sort-imports
+import readme from './README.stories.mdx';
+import { CTA_TYPE } from '../../cta/defs';
+import { ICON_PLACEMENT } from '../../link-with-icon/link-with-icon';
+
+const hrefsForType = {
+  [CTA_TYPE.LOCAL]: 'https://www.example.com',
+  [CTA_TYPE.EXTERNAL]: 'https://www.example.com',
+  [CTA_TYPE.DOWNLOAD]: 'https://www.ibm.com/annualreport/assets/downloads/IBM_Annual_Report_2019.pdf',
+  [CTA_TYPE.VIDEO]: '1_9h94wo6b',
+};
+
+const knobNamesForType = {
+  [CTA_TYPE.LOCAL]: 'Content link href (href)',
+  [CTA_TYPE.EXTERNAL]: 'Content link href (href)',
+  [CTA_TYPE.DOWNLOAD]: 'Download link href (href)',
+  [CTA_TYPE.VIDEO]: 'Video ID (href)',
+};
+
+const types = {
+  [`Local (${CTA_TYPE.LOCAL})`]: CTA_TYPE.LOCAL,
+  [`External (${CTA_TYPE.EXTERNAL})`]: CTA_TYPE.EXTERNAL,
+  [`Download (${CTA_TYPE.DOWNLOAD})`]: CTA_TYPE.DOWNLOAD,
+  [`Video (${CTA_TYPE.VIDEO})`]: CTA_TYPE.VIDEO,
+};
+
+export const Default = ({ parameters }) => {
+  const { heading, ctaType, download, href, iconPlacement = ICON_PLACEMENT.RIGHT } = parameters?.props?.ContentGroupBanner ?? {};
+  return !ctaType
+    ? html`
+        <dds-content-group-banner>
+          <dds-content-group-heading>${heading}</dds-content-group-heading>
+
+          <dds-link-list type="vertical" slot="complementary">
+            <dds-link-list-item icon-placement="${iconPlacement}" href="https://example.com">
+              Learn more about Kubernetes ${ArrowRight20({ slot: 'icon' })}
+            </dds-link-list-item>
+            <dds-link-list-item icon-placement="${iconPlacement}" href="https://example.com">
+              Containerization A Complete Guide ${ArrowRight20({ slot: 'icon' })}
+            </dds-link-list-item>
+          </dds-link-list>
+        </dds-content-group-banner>
+      `
+    : html`
+        <dds-content-group-banner>
+          <dds-content-group-heading>${heading}</dds-content-group-heading>
+
+          <dds-link-list type="vertical" slot="complementary">
+            <dds-link-list-item-cta
+              icon-placement="${iconPlacement}"
+              href="${ifNonNull(href)}"
+              cta-type="${ifNonNull(ctaType)}"
+              download="${ifNonNull(download)}"
+            >
+              Learn more about Kubernetes
+            </dds-link-list-item-cta>
+            <dds-link-list-item-cta
+              icon-placement="${iconPlacement}"
+              href="${ifNonNull(href)}"
+              cta-type="${ifNonNull(ctaType)}"
+              download="${ifNonNull(download)}"
+            >
+              Containerization A Complete Guide
+            </dds-link-list-item-cta>
+          </dds-link-list>
+        </dds-content-group-banner>
+      `;
+};
+
+export default {
+  title: 'Components/Content group banner',
+  decorators: [
+    story => html`
+      <dds-video-cta-container class="dds-ce-demo-devenv--simple-grid dds-ce-demo-devenv--simple-grid--content-group-banner">
```

Comment:
A bit confused that the wrapper here is `dds-video-cta-container`...

Reply:
@jeffchew The `<dds-video-cta-container>` is needed in order for the video CTA type to open as an overlay. I'm just following the example shown here ( see - https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/link-list/__stories__/link-list.stories.ts#L268). Also, I can wrap it around the content group banner component as well.

```
     <dds-video-cta-container>
          <dds-content-group-banner>
            <dds-content-group-heading>${heading}</dds-content-group-heading>
            
            <dds-link-list type="vertical" slot="complementary">
              <dds-link-list-item-cta
                icon-placement="${iconPlacement}"
                href="${ifNonNull(href)}"
                cta-type="${ifNonNull(ctaType)}"
                download="${ifNonNull(download)}"
              >
                Learn more about Kubernetes
              </dds-link-list-item-cta>
              <dds-link-list-item-cta
                icon-placement="${iconPlacement}"
                href="${ifNonNull(href)}"
                cta-type="${ifNonNull(ctaType)}"
                download="${ifNonNull(download)}"
              >
                Containerization A Complete Guide
              </dds-link-list-item-cta>
            </dds-link-list>
          </dds-content-group-banner>
        </dds-video-cta-container>
```

Reply:
That's fine, think we can leave as-is, thanks!